api/google.go

```diff
@@ -1,4 +1,5 @@
 // Copyright 2019 Booking.com
+// Copyright 2019 Google LLC
```

Comment:
That's concerning. I'm afraid this change needs an approval of Booking.com legals.